README.md

```diff
@@ -60,8 +60,8 @@ Bluejay's primary goals are:
 
 ## Requirements
 
-- iOS 10 or above
-- Xcode 10.2.1 or above
+- iOS 11 or above
+- Xcode 11.3.1 or above
```

Comment:
This says iOS 11, but it looks like the deployment target is still iOS 10, should we bump the deployment target as well, does it actually not work on iOS 10 any more with this change?

Also, we might want to just check if it does still build on Xcode 10 with this change, and if to maybe say `Xcode 10.2.1 or later (Xcode 11.3.1 or later recommended)`

Reply:
Thanks and good catch @nbrooke! I've bumped the deployment target to 11.

Here are some of my strategies and thinking (there may be many things I've overlooked, so please let me know if so):
- iOS 10 adoption is at around 0.7% right now according to https://david-smith.org/iosversionstats/
  - We will probably never test on another iOS 10 device again (also not sure if it's worth it)
  - Might as well not promise Bluejay will work on iOS 10 if we aren't consistently using, testing, nor developing against iOS 10
- I am thinking similarly for Xcode, and am not sure if there's a good reason to guarantee support for Xcode 10.2.1.
  - I think at around WWDC or the autumn season when a new Xcode is potentially very buggy, then it makes sense to support two Xcode versions then, but right now Xcode 11 is **probably** pretty stable

In general, I am considering the amount of resources we have available to dedicate to Bluejay, and at least for now, limiting the dev environment (iOS and Xcode versions) to what we will most likely use on a daily basis seems like a good idea...I think!

Reply:
Yeah, that's fine as long as we are consistant.